gulpfile.js

```diff
@@ -1,93 +1,59 @@
 async function getContributors(user, repo, maxCount = 1) {
-  const contributors = (await axios.get(
-    `https://api.github.com/repos/${encodeURIComponent(user)}/${encodeURIComponent(repo)}/contributors`,
-    { params: { per_page: maxCount } }
-  )).data;
-
-  return Promise.all(contributors.map(async (contributor)=> {
-    return {...contributor, ...(await axios.get(
-      `https://api.github.com/users/${encodeURIComponent(contributor.login)}`
-    )).data};
-  }))
+  const contributors = (await axios.get(`https://api.github.com/repos/${user}/${repo}/contributors`, { params: { per_page: maxCount } })).data;
+  return Promise.all(contributors.map(async contributor => ({
+    ...contributor,
+    ...(await axios.get(`https://api.github.com/users/${contributor.login}`)).data
+  })));
 }
 
-const packageJSON = gulp.task('package', async function () {
-  const CONTRIBUTION_THRESHOLD = 3;
-
-  const npm = JSON.parse(await fs.readFile('package.json'));
-
+// Task to update package.json with contributors
+gulp.task('package', async () => {
+  const npm = await fs.readJson('package.json');
   try {
     const contributors = await getContributors('axios', 'axios', 15);
-
     npm.contributors = contributors
-      .filter(
-        ({type, contributions}) => type.toLowerCase() === 'user' && contributions >= CONTRIBUTION_THRESHOLD
-      )
-      .map(({login, name, url}) => `${name || login} (https://github.com/${login})`);
-
-    await fs.writeFile('package.json', JSON.stringify(npm, null, 2));
+      .filter(({ type, contributions }) => type.toLowerCase() === 'user' && contributions >= 3)
+      .map(({ login, name }) => `${name || login} (https://github.com/${login})`);
+    await fs.writeJson('package.json', npm, { spaces: 2 });
   } catch (err) {
-    if (axios.isAxiosError(err) && err.response && err.response.status === 403) {
-      throw Error(`GitHub API Error: ${err.response.data && err.response.data.message}`);
+    if (axios.isAxiosError(err) && err.response.status === 403) {
+      throw new Error(`GitHub API Error: ${err.response.data.message}`);
     }
     throw err;
   }
 });
 
-const env = gulp.task('env', async function () {
-  var npm = JSON.parse(await fs.readFile('package.json'));
-
+// Task to write environment variables
+gulp.task('env', async () => {
+  const npm = await fs.readJson('package.json');
   const envFilePath = './lib/env/data.js';
-
-  await fs.writeFile(envFilePath, Object.entries({
-    VERSION: (argv.bump || npm.version).replace(/^v/, '')
-  }).map(([key, value]) => {
-    return `export const ${key} = ${JSON.stringify(value)};`
-  }).join('\n'));
+  const content = `export const VERSION = ${JSON.stringify((argv.bump || npm.version).replace(/^v/, ''))};`;
+  await fs.writeFile(envFilePath, content);
 });
 
-const version = gulp.series('bower', 'env', 'package');
+// Version task: runs bower, env, and package
+gulp.task('version', gulp.series('bower', 'env', 'package'));
 
-export {
```

Comment:
https://gulpjs.com/docs/en/api/task/
`gulp.task` is deprecated, so if we are going to update the pipeline, it would be better to migrate all tasks to the new syntax by exporting a pure task function